lib/dbadaptor.py

```diff
@@ -124,13 +131,13 @@ def select(self, conn, table_name, cols='*', where=None, orderby=None, **kwargs)
             sql += ' WHERE %s' % where
         if orderby is not None:
             sql += ' ORDER BY %s'(','.join(orderby))
-        c.execute(sql)
-        data = list(c)
+        with closing(conn.cursor()) as c:
+            c.execute(sql)
+            data = c.fetchall()
         return data
 
-    def update(self, conn, table_name, values, where, ph):
+    def update(self, conn, table_name, values, ph, where=None):
```

Comment:
Is there any reason to change the postion of arguments? 
And in dbadaptor I would not set a default value for the argument, it's duplicated with the method in pysixdb. In dbadaptor, especially for `update` and `delete` method, I don't set a default value for `where` argument is to make it mandatory to pass a `where` condition when calling these methods, bescause it is dangerous to call these methods without filtered condition, it will change all the records in database.